migrations/20170731211519_messages.js

```diff
@@ -0,0 +1,23 @@
+exports.up = knex =>
+  knex.schema.createTable('messages', (table) => {
+    table.increments('message_id').primary();
+    table
+      .string('user_id')
```

Comment:
I'm interested to know why the user_id became a string and would love to hear you're reasoning - in person if i'm so lucky

Reply:
Answer! the user ID comes from slack as a string (combo of ints and letters) so its' a more straightforward implementation to just use the provided string as a foreign key. According to kurtis and tyler, we have no current use for a sequential key, as user_id order doesn't matter.